pkg/gofr/grpc_test.go

```diff
@@ -51,3 +53,23 @@ func TestGRPC_ServerRun(t *testing.T) {
 		assert.Contains(t, out, tc.expLog, "TEST[%d], Failed.\n", i)
 	}
 }
+
+func TestGRPC_ServerShutdown(t *testing.T) {
+	c := container.Container{
+		Logger: logging.NewLogger(logging.DEBUG),
+	}
+
+	g := newGRPCServer(&c, 9999)
+
+	var errChan = make(chan error, 1)
+	go func() {
+		time.Sleep(2 * time.Second)
```

Comment:
2 seconds is too long for tests

Reply:
This is probably leftovers from previous contributor. Feel free to fix it up if you have spare time today

Reply:
I'm opening an issue then

Reply:
I created
- https://github.com/gofr-dev/gofr/issues/795